larsim/PhotonPropagation/PDFastSimPAR_module.cc

```diff
@@ -465,51 +528,122 @@ namespace phot {
       double edeposit = edepi.Energy() / nphot;
       double pos[3] = {edepi.MidPointX(), edepi.MidPointY(), edepi.MidPointZ()};
       geo::Point_t const ScintPoint = {pos[0], pos[1], pos[2]};
+      G4ThreeVector ScintPoint_v = {pos[0], pos[1], pos[2]};
```

Comment:
Can the interface of `propagationTime(...)` be adjusted so this extra variable is not necessary?